concordium-sdk/src/main/java/com/concordium/sdk/responses/blocksummary/updates/chainparameters/ChainParameters.java

```diff
@@ -0,0 +1,81 @@
+package com.concordium.sdk.responses.blocksummary.updates.chainparameters;
+
+import com.concordium.sdk.responses.blocksummary.updates.Fraction;
+import com.concordium.sdk.responses.blocksummary.updates.chainparameters.rewards.RewardParameters;
+import com.concordium.sdk.transactions.CCDAmount;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@ToString
+public final class ChainParameters {
+    private final CCDAmount minimumThresholdForBaking;
+    private final RewardParameters rewardParameters;
+    private final Fraction microGTUPerEuro;
+    private final int foundationAccountIndex;
+    private final int accountCreationLimit;
+    private final int bakerCooldownEpochs;
+    private final double electionDifficulty;
+    private final Fraction euroPerEnergy;
+    private final double mintPerPayday;
+    private final int poolOwnerCooldown;
+    private final double capitalBound;
+    private final int rewardPeriodLength;
+    private final double passiveBakingCommission;
+    private final Fraction leverageBound;
+    private final double passiveFinalizationCommission;
+    private final int delegatorCooldown;
+    private final double passiveTransactionCommission;
+    private final Range bakingCommissionRange;
+    private final Range finalizationCommissionRange;
+    private final Range transactionCommissionRange;
+    private final CCDAmount minimumEquityCapital;
+
+    @JsonCreator
+    ChainParameters(
```

Comment:
This type should accommodate the two different versions. Not just putting everything into one large blob.

Reply:
And doc